Fixes coder example and adds more explanation (#749)
pupil-labs/pupil-docs: neon/data-collection/psychopy/index.md

## Coder

> The PsychoPy team doesn't recommend using Coder for eyetracking experiments, and suggests instead that users employ Builder instead. Of course, Builder doesn't suit all needs, so please keep in mind that the [Realtime Python API](https://docs.pupil-labs.com/neon/real-time-api/tutorials/) is fully usable within PsychoPy, as is the [real-time screen gaze package](https://github.com/pupil-labs/real-time-screen-gaze). Some users may find these APIs easier to work with than PsychoPy's common eyetracker interface, but using the Realtime API directly won't provide gaze to PsychoPy's data logging or eyetracking components (e.g., areas-of-interest). If you have need gaze data fully-integrated with PsychoPy's software stack and need to use Coder, then you'll need to configure ioHub, add AprilTag markers to the screen, and register the screen surface with the eyetracker. The example below shows how to collect and visualize realtime gaze position and pupil diameter in PsychoPy Coder.

Reviewer: ```suggestion
The PsychoPy team doesn't recommend using Coder for eyetracking experiments and suggests that users employ Builder instead. Of course, Builder doesn't suit all needs, so please keep in mind that the [Realtime Python API](https://docs.pupil-labs.com/neon/real-time-api/tutorials/) is fully usable within PsychoPy, as is the [real-time screen gaze package](https://github.com/pupil-labs/real-time-screen-gaze). Some users may find these APIs easier to work with than PsychoPy's common eyetracker interface, but using the Realtime API directly won't provide gaze to PsychoPy's data logging or eyetracking components (e.g., areas-of-interest). If you need gaze data fully integrated with PsychoPy's software stack and need to use Coder, then it will be necessary to configure ioHub, add AprilTag markers to the screen, and register the screen surface with the eyetracker. The example below shows how to collect and visualize real-time gaze position and pupil diameter in PsychoPy Coder.
```
